hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/BlockOutputStreamEntry.java

```diff
@@ -47,7 +48,7 @@
 public class BlockOutputStreamEntry extends OutputStream {
 
   private final OzoneClientConfig config;
-  private OutputStream outputStream;
+  private RatisBlockOutputStream outputStream;
```

Comment:
Yes, we can keep using `OutputStream` here.

BTW, ECBlockOutputStream seems not using `outputStream` at all since it is always set to a `RatisBlockOutputStream` object.